src/app/components/NavBar.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { SFC } from 'react';
+import styled from 'styled-components';
+import openstaxLogo from '../../assets/logo.svg';
+import theme from '../theme';
+import typography from '../components/Typography';
+
+const TopBar = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 5rem;
+  max-width: 117rem;
+  background: ${theme.color.neutral.base};
+  margin: 0 auto;
+
+  @media (max-width: 48em) {
+    height: 3.6rem;
+  }
+`;
+
+const LogoLink = styled.a`
+`;
+
+const HeaderImage = styled.img`
+  width: auto;
+  height: 3rem;
+
+  @media (max-width: 48em) {
+    height: 2rem;
+  }
+`;
+
+const LoginTxt = styled.a`
```

Comment:
this button kinda twitches up when you hover, maybe you need some negative margin on the hover state to compensate for the border?